pyproject.toml

```diff
@@ -60,6 +60,7 @@ doc = [
     "sphinx>=7.2.0",
     "autodoc-pydantic>=2.0.0",
     "ansys-sphinx-theme>=0.12.5",
+    "sphinx-gallery",
```

Comment:
You need to run the command `pdm lock` so that the dependency is added to the project (this causes the build error on the CI)